### DISCUSSION TITLE
Mellanox nic use commit 4bd064d ipxe.efi will happen DHCPDISCOVER failed,but use the latest master ipxe.efi will work ok

### DISCUSSION THREAD

**Discussion by xcc-123** [ipxe/ipxe#1248] [Q&A]
Mellanox nic use commit 4bd064d ipxe.efi will happen DHCPDISCOVER failed,
![微信图片_20240630002649](https://github.com/ipxe/ipxe/assets/68419870/69e3974c-49d8-4f29-a4d6-b3c9afcea64e)

but use the latest master ipxe.efi will work ok
![微信图片_20240630002656](https://github.com/ipxe/ipxe/assets/68419870/14e0d6e7-0ce0-448f-9215-e935666145ce)

if i let nic paramter KEEP_ETH_LINK_UP_P1=TRUE,it also can use commit 4bd064d ipxe.efi,it will work ok
but set KEEP_ETH_LINK_UP_P1=FALSE,i need use the latest master ipxe.efi  which can work.

why happen DHCPDISCOVER failed？ which possible commit can solve it?

**Comment by NiKiZe** [comment]
Duplicate of #1247 
As written there, use git bisect if you need to find out.
We will not give support here for anything other than current master.
